packages/core/src/validate.ts

```diff
@@ -447,3 +447,15 @@ function* getEnumErrors(contractDef: ContractDefinition, decodeSrc: SrcDecoder):
     };
   }
 }
+
+let silenced = false;
+
+export function silenceWarnings(): void {
+  console.error(
+    '\n' +
+      chalk.keyword('orange').bold('Warning: ') +
+      `All subsequent Upgrades warnings will be silenced.\n\n` +
```

Comment:
Maybe warning in here that this message shouldn't be seen outside testing envs?

Reply:
This was not a bad idea, but I decided not to add it in the message because I don't feel strongly about only using this in testing environments. I see it just as a recommendation.